schema.sql

```diff
@@ -88,4 +88,23 @@ CREATE TABLE visits(
     CONSTRAINT vet_id_fk FOREIGN KEY(vets_id) REFERENCES vets(id)
 );
 
-COMMIT;
\ No newline at end of file
+COMMIT;
+
+-- Update added emails column to owners table
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+-- This will add 3.594.280 visits considering you have 10 animals, 4 vets, and it will use around ~87.000 timestamps (~4min approx.)
+INSERT INTO visits (animal_id, vet_id, date_of_visit) SELECT * FROM (SELECT id FROM animals) animal_ids, (SELECT id FROM vets) vets_ids, generate_series('1980-01-01'::timestamp, '2021-01-01', '4 hours') visit_timestamp;
+
+
+-- This will add 2.500.000 owners with full_name = 'Owner <X>' and email = 'owner_<X>@email.com' (~2min approx.)
+insert into owners (full_name, email) select 'Owner ' || generate_series(1,2500000), 'owner_' || generate_series(1,2500000) || '@mail.com';
+
+-- We create  an index feature for the owner_email column of owners table;
+CREATE INDEX owner_email ON owners(email ASC);
+
+-- We create  an index feature for the visits_index column of visits table
+CREATE INDEX visits_index ON visits(id ASC);
+
+-- We create  an index feature for the visits_index column of visits table for the animals_id
+CREATE INDEX visits_index ON visits(animals_id ASC);
```

Comment:
- [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_performance_audit.md) for this project, '' `While you are making changes, check if they help by running EXPLAIN ANALYZE. Once you succeed, take a screenshot of the EXPLAIN ANALYZE result showing that time actually decreased.` ", " `In your pull request description include screenshots with EXPLAIN ANALYZE results (before and after you improve the speed) for each of the 3 problematic queries.` ", please kindly include a pairs of screenshots in the PR description with the results of the  EXPLAIN ANALYZE command for each of the queries (in total it should be 6 screenshots).